agent/src/flow_generator/protocol_logs/http.rs

```diff
@@ -608,6 +612,7 @@ impl L7ProtocolParserInterface for HttpLog {
                 };
                 info.endpoint = Some(handle_endpoint(config, path));
             }
+            self.wasm_hook(param, payload, &mut info);
```

Comment:
`wasm_hook` should be moved before L604, before endpoint rewriting

Reply:
In uprobe mode (when `param.ebpf_type == EbpfType::GoHttp2Uprobe`), headers are reported in a way different from other modes: one payload contains one header. Calling wasm plugin on every payload would be wasted effort. We can skip calling plugins under this condition for now, or check if we have enough information (or headers) before calling plugins.

Reply:
> In uprobe mode (when `param.ebpf_type == EbpfType::GoHttp2Uprobe`), headers are reported in a way different from other modes: one payload contains one header. Calling wasm plugin on every payload would be wasted effort. We can skip calling plugins under this condition for now, or check if we have enough information (or headers) before calling plugins.

The `extra-log-fields` configuration is enough to collect header fields, while the wasm plugin is mainly used to parse the data. Therefore, in the case where param.ebpf_type == EbpfType::GoHttp2Uprobe, the call to the wasm plugin has been skipped.

Reply:
When uprobe for grpc is enabled, kprobe for grpc will be disabled automatically.

So we should also handle uprobe case， but in this PR we can just leave a TODO and fix it in the future.